P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1330 @@
+                <person role="user" sex="female" xml:id="cmc_user_06">
+                  <persName type="userName">P</persName>
+                  <note type="link">/wiki/User:P</note>
+                  <affiliation>
+                    <email>pat@super.net</email>
+                    <country>ES</country>
+                  </affiliation>
+                </person>
+                <person role="user" xml:id="cmc_user_07">
+                  <persName type="userName">PKP</persName>
+                  <note type="link">/wiki/User:Pi</note>
+                </person>
+              </listPerson>
+            </particDesc>
+          </profileDesc>
+          <!-- In the <body>: -->
+          <div type="wiki_discussion_page" n="073">
+            <!-- 4 other <post>s -->
+            <post modality="written" xml:id="cmc_post04" indentLevel="1" replyTo="#cmc_post_073.004"
+              who="#cmc_user_06">
+              <p>Those haven't happened. If they do, we can revisit the concern.</p>
+              <signed generatedBy="template" rend="noLineBreak">
+                <ref target="/wiki/User:P">P</ref>
+                <date>01:35, 8 April 2014 (UTC)</date>
+              </signed>
+            </post>
+          </div>
+        </egXML>
+      </p>
+      <p>In the following version of the <gi>body</gi> portion of the same example, the list of
+        interactants is stored in a separate file (in this case the file <name type="file"
+          >userList.xml</name> in the same directory). <egXML
+          xmlns="http://www.tei-c.org/ns/Examples" source="#BIB_WPTalkAstronomicalObject"
+          xml:lang="en">
+          <!-- In the <body>: -->
+          <div type="wiki_discussion_page" n="073">
+            <!-- 4 other <post>s -->
+            <post modality="written" xml:id="cmc_post05" indentLevel="1" replyTo="#cmc_post_073.004"
+              who="./userList.xml#cmc_user_06">
+              <p>Those haven't happened. If they do, we can revisit the concern.</p>
+              <signed generatedBy="template" rend="noLineBreak">
+                <ref target="/wiki/User:P">P</ref>
+                <date>01:35, 8 April 2014 (UTC)</date>
+              </signed>
+            </post>
+          </div>
+        </egXML> Alternatively, a <gi>prefixDef</gi> may be used to declare a prefix which can be
+        used in the value of <att>who</att> to generate a complete URI, thus making the values of
+          <att>who</att> shorter, less error-prone, and easier to maintain. For example, the prefix
+          <code>uL:</code> could be used to map the value <val>uL:06</val> to
+          <code>file:/userList.xml#cmc_user_06</code>. See <ptr target="#SAPU"/> for more
+        information on establishing prefix definitions.</p>
+      <p>This indirection — using a <gi>listPerson</gi>, particularly one in a separate file, to
+        store information about the users involved in a CMC interaction — is particularly useful
```

Comment:
```suggestion
      <p>This indirection—using a <gi>listPerson</gi>, particularly one in a separate file, to
        store information about the users involved in a CMC interaction—is especially useful
```

Reply:
Amended all em dashes for consistency in https://github.com/TEIC/TEI/pull/2537/commits/9407cd7f6a866ec8c7df720685569d12b39bbfc5